ansible/roles/openregister_config/templates/openregister-config.yaml.j2

```diff
@@ -66,8 +66,9 @@ server:
       port: {% if paas %}${PORT}{% else %}8080{% endif %}
   registerDefaultExceptionMappers: false
   requestLog:
+    type: classic
     appenders:
-      - type: access-logstash-console
```

Comment:
This change should only be applied when non running on PaaS. There's a magical `paas` bool you can use to make the generated configs different.